thorlcr/master/thgraphmanager.cpp

```diff
@@ -1328,7 +1328,7 @@ void thorMain(ILogMsgHandler *logHandler, const char *wuid, const char *graphNam
             else
             {
                 unsigned lingerPeriod = globals->getPropInt("@lingerPeriod", DEFAULT_LINGER_SECS)*1000;
```

Comment:
Is the default linger time suitable?

Reply:
for a contested cluster it is.. e.g. inter-leaved graphs from queues jobs
would be enough e.g. for regression suite.

I've upped default to 1 min